pkg/controller/metadata/metadata_controller.go

```diff
@@ -216,7 +216,13 @@ func (r *ReconcileMetadata) generateMetadataSecretData(instance *verifyv1beta1.M
 	if err != nil {
 		return nil, fmt.Errorf("generateMetadataSecretData: %s", err)
 	}
-	metadataCACertsConcat := bytes.Join(metadataCACerts, []byte("\n"))
+	var metadataCACertsConcat []byte
+	if len(metadataCACerts) > 1 {
+		metadataCACertsConcat = bytes.Join(metadataCACerts[1:], []byte("\n"))
+	} else {
+		metadataCACertsConcat = metadataCACerts[0]
```

Comment:
We don't check for len 0  as this would result in an `err` above.

Reply:
How do we know the one we want is always at the same index?

Reply:
The cert chain is generated leaf (idx 0) -> intermediate (idx 1) -> root (idx 2)
We know we need to drop idx 0 or take 0  if that's the only one